src/sgame/sg_struct.h

```diff
@@ -25,7 +25,7 @@ along with Unvanquished Source Code.  If not, see <http://www.gnu.org/licenses/>
 #ifndef SG_STRUCT_H_
 #define SG_STRUCT_H_
 
-#include "sg_entities.h"
+#include "sg_local.h"
```

Comment:
sg_local.h includes sg_struct.h so this doesn't seem good

Reply:
fixed